keyboards/crkbd/keymaps/mcrown/keymap.c

```diff
@@ -0,0 +1,156 @@
+          _____________________ADJUST_L1______________________,      _____________________ADJUST_R1______________________,
+          /*|--------+--------+--------+--------+--------+----|     |--------+--------+--------+--------+--------+-----------|*/
+          _____________________ADJUST_L2______________________,      _____________________ADJUST_R2______________________,
+          /*|--------+--------+--------+--------+--------+----|     |--------+--------+--------+--------+--------+-----------|*/
+          _____________________ADJUST_L3______________________,      _____________________ADJUST_R3______________________,
+          /*|--------+--------+--------+--------+--------+----|     |--------+--------+--------+--------+--------+-----------|*/
+                            _____________MOD_LEFT_____________,      _____________MOD_RIGHT____________
+                        /*|-----------------------------------|     |----------------------------------|*/
+  ),
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record)
+{
+    bool user_records_press=TRUE;
+
+    if (record->event.pressed)
+    {
+        add_keylog(keycode);
+    }
+
+    switch (keycode)
+    {
+        case QWERTY:
+            if (record->event.pressed)
+            {
+                set_single_persistent_default_layer(_QWERTY);
+            }
+            break;
+
+        case LOWER:
+            if(record->event.pressed)
+            {
+                layer_on(_LOWER);
+                update_tri_layer(_LOWER, _RAISE, _ADJUST);
+            }
+            else
+            {
+                layer_off(_LOWER);
+                update_tri_layer(_LOWER, _RAISE, _ADJUST);
+            }
+            break;
+
+        case RAISE:
+            if (record->event.pressed)
+            {
+                layer_on(_RAISE);
+                update_tri_layer(_LOWER, _RAISE, _ADJUST);
+            }
+            else
+            {
+                layer_off(_RAISE);
+                update_tri_layer(_LOWER, _RAISE, _ADJUST);
+            }
+            break;
+
+        case ADJUST:
+            if (record->event.pressed)
+            {
+                layer_on(_ADJUST);
+            }
+            else
+            {
+                layer_off(_ADJUST);
+            }
+            break;
+
+        case RGB_MOD:
+            #ifdef RGBLIGHT_ENABLE
+            if(record->event.pressed)
+            {
+                rgblight_mode(RGB_current_mode);
+                rgblight_step();
+                RGB_current_mode = rgblight_config.mode;
+            }
+            #endif
+            break;
+
+        case RGBRST:
+            #ifdef RGBLIGHT_ENABLE
+            if(record->event.pressed)
+            {
+                eeconfig_update_rgblight_default();
+                rgblight_enable();
+                RGB_current_mode = rgblight_config.mode;
+            }
+            #endif
+            break;
+
+        default:
+            user_records_press=FALSE;
```

Comment:
Why the `FALSE` over `false`?

Reply:
I redefine these in mcrown.h file. Since bool types are not part of the Standard C, I considered that their values  ```TRUE``` and ```FA:SE``` should be capitalized to identify it is a defininition.

Reply:
Well, they may not be standard C, but they appear to be supported by avr-gcc and arm gcc, and would prefer that the lower case version be used.

Specifically, I would prefer to keep the [C coding style](https://docs.qmk.fm/#/coding_conventions_c) for the project (including the rest of yours files, actually).

Reply:
Done!